Dockerfile

```diff
@@ -15,8 +15,10 @@ COPY ./RTL /RTL
 
 ADD ./docker_entrypoint.sh /usr/local/bin/docker_entrypoint.sh
 RUN chmod +x /usr/local/bin/docker_entrypoint.sh
-ADD ./configurator/target/armv7-unknown-linux-musleabihf/release/configurator /usr/local/bin/configurator
+ADD ./configurator/target/aarch64-unknown-linux-musl/release/configurator /usr/local/bin/configurator
 RUN chmod +x /usr/local/bin/configurator
+ADD ./config.sh /usr/local/bin/config
+RUN chmod a+x /usr/local/bin/config
```

Comment:
these steps can be removed in favor of using the `config` command from the `compat` image